script/releases/v1.3.0-slashing/1-deployContracts.s.sol

```diff
@@ -0,0 +1,552 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {EOADeployer} from "zeus-templates/templates/EOADeployer.sol";
+import "../Env.sol";
+
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/**
+ * Purpose: use an EOA to deploy all of the new contracts for this upgrade.
+ */
+contract Deploy is EOADeployer {
+    using Env for *;
+
+    function _runAsEOA() internal override {
+        vm.startBroadcast();
+
+        /// permissions/
+
+        address[] memory pausers = new address[](3);
+        pausers[0] = Env.pauserMultisig();
+        pausers[1] = Env.opsMultisig();
+        pausers[2] = Env.executorMultisig();
+
+        deployImpl({
+            name: type(PauserRegistry).name,
+            deployedTo: address(new PauserRegistry({_pausers: pausers, _unpauser: Env.executorMultisig()}))
```

Comment:
Thoughts on making the unpauser `Env.opsMultisig()`? It would be good to keep the barrier to unpausing low

Reply:
See comment below - holding off on this.